simulations/.env.shared

```diff
@@ -0,0 +1,30 @@
+# Node Registry Configuration
```

Comment:
This file can be removed now as configurations of simulation is added to the json config file.